### ISSUE TITLE
Report and Download issue

### ISSUE THREAD

**Issue by Rudy-Weak** [Vulnerator/Vulnerator#218]
#### Prerequisites
_Before submitting a new issue, please ensure you have completed the following (delete the space in the box and replace it with an "x" to denote that it has been completed)_

- [x] I have ensured that I am running the latest release
- [x] The issue is repeatable
- [x] The issue has not already been reported

#### Description
When I download the File for the latest release the options for creating reports did not come with the download. Confirmed with other members of my team that were working with the 7.0 alpha release that they had options for creating reports which were blank for me. it seems to only do this if you don't export the files. that might be the reason it is not working but it could be a issues

#### To Reproduce
all I did was download the latest release and did not extract it from the zip file before running the tool. it did not have the reporting options.

#### Expected Outcome
when I downloaded the file I expected the tool to be like the previous version and work just fine.

#### Actual Outcome
I could not figure out how to generate reports and confirmed with another person working with the tool on the same version that they were not having the same issue

#### System Information
 - _**OS:**_ [Windows 10]
 - _**Version:**_ [v7.0.0 Alpha]

#### Additional Context
downloaded a fresh version after deleting all copies of it from my system and when i download the fresh version it had the options for creating the reports.

#### Attachments
_Please provide any relevant attachments, as you see fit (e.g. screenshots); if supplying vulnerability data (e.g. CKL/Nessus files or reports), please ensure that they are sanitized of IP addresses and host names and **email** them to alex.kuchta@navy.mil - **DO NOT POST VULNERABILITY FILES HERE**_

- [x] Vulnerator Log (required for bugs) - this can be found at ```C:\Users\%UserName%\AppData\Roaming\Vulnerator\V6Log.txt```, where ```%UserName%``` is the user profile used to run the application
[V6Log.txt](https://github.com/Vulnerator/Vulnerator/files/6223546/V6Log.txt)
- [x] Screenshots (optional, _**SANITIZED/REDACTED**_)
![Bug Report Vulnerator](https://user-images.githubusercontent.com/81582573/112877744-bc113e80-908c-11eb-9788-b099760c0767.JPG)
- [ ] Vulnerability Files (optional, _**SANITIZED AND EMAILED**_)

**Comment by amkuchta** [comment]
@Rudy-Weak sorry for the late response! Out of curiosity, did you try closing and reopening the application?

**Comment by Rudy-Weak** [comment]
yeah, and it had no effect. still having the problem of opening the file from the zip folder.
